CSS/aside.css

```diff
@@ -1,5 +1,13 @@
-.aside {
+.aside-container {
     grid-area: aside;
+    position: sticky;
```

Comment:
Why sticky? Why not fixed?

Reply:
I don't want it to come out of the normal flow of the document.